src/fixtures/listed.tokenlist.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
I'd rather this fixtures folder wasn't as high level, e.g. right under /src. Maybe best to nest with /src/lib for now?

Reply:
how about top level src/test folder? there is `__mocks__` which could go to src/test/mocks as well?

Reply:
Yeah, I like that but I'm worried that there is some auto-magic going on with that mocks folder. Could you try moving it and run the tests to see if they still pass?